examples/circuitplayground_light_neopixels.py

```diff
@@ -13,9 +13,9 @@
 
 
 def scale_range(value):
-    """Scale a value from 0-320 (light range) to 0-10 (the number of NeoPixels).
+    """Scale a value from 0-320 (light range) to 0-9 (NeoPixel range).
     Allows remapping light value to pixel position."""
-    return int(value / 320 * 10)
+    return int(value / 320 * 9)
```

Comment:
You'll want to use `round` instead of `int` so it is more evenly distributed.

Reply:
Ah I missed that. Updated.